### ISSUE TITLE
Typescript `declaration file` error

### ISSUE THREAD

**Issue by harleylang** [fvilers/disable-react-devtools#4]
Thank you for preparing this library, it is so handy. I heard about it on the Syntax.Fm podcast.

I'm checking-in because I noticed with typescript that the module throws an import error:

```
Could not find a declaration file for module '@fvilers/disable-react-devtools'. '/react-app/node_modules/@fvilers/disable-react-devtools/index.js' implicitly has an 'any' type.
  Try `npm install @types/fvilers__disable-react-devtools` if it exists or add a new declaration (.d.ts) file containing `declare module '@fvilers/disable-react-devtools';`ts(7016)
```

I was able to solve this issue with the following code in my `.d.ts` file:

```
declare module '@fvilers/disable-react-devtools' {
    export function disableReactDevTools(): () => void;
};
```

Would you accept a PR with this codified into an `index.d.ts` file? I'm not sure if that is the standard manner for typing a library, so if there is a better way please let me know and I'd be happy to send a quick PR.

**Comment by fvilers** [comment]
Hi @harleylang ! Thanks for your message. I'd gladly accept this as a PR. It is the standard manner to provide a typing library. I prefer to embed the declaration file within the library itself than with a different NPM package.

**Comment by harleylang** [comment]
Awesome, happy to help! Pull request sent, let me know in the revisions if anything else is needed to make the module typescript compliant.

**Comment by fvilers** [comment]
Thanks again for your help, I published version 1.2.0 with the definition file.
